src/main/resources/config/liquibase/changelog/20200406120000_upload_validation_column.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+    <changeSet author="bmatern" id="addValidationColumn">
+        <addColumn
+                   tableName="upload">
+            <column name="validation_feedback" type="varchar(255)" defaultValue=""/>
+        </addColumn>
+    </changeSet>
+
```

Comment:
Here you may want to add the new role to the jhi_authority table

Reply:
This has been done, ROLE_VALIDATION